src/styles/blocks/animations.scss

```diff
@@ -0,0 +1,9 @@
+@keyframes move {
```

Comment:
Consider renaming the '@keyframes move' to a more descriptive name, like '@keyframes rotate360'.